runpod_worker/utils.py

```
@@ -0,0 +1,284 @@
+        image_path: Path to image file
+        
+    Returns:
+        Base64 encoded image data
+    """
+    try:
+        with open(image_path, 'rb') as image_file:
+            image_data = image_file.read()
+            base64_string = base64.b64encode(image_data).decode('utf-8')
+            return base64_string
+    except Exception as e:
+        raise ValueError(f"Failed to encode image to base64: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)